os/tools/check_package_size.py

```diff
@@ -113,8 +113,9 @@ def check_binary_header(bin_type):
 COMMON_PART_TMP_SIZE = FLASH_SIZE
 APP1_PART_TMP_SIZE = FLASH_SIZE
 APP2_PART_TMP_SIZE = FLASH_SIZE
-
+size=len(SIZE_LIST)
 for name in NAME_LIST :
```

Comment:
This error is because of the kernel not found in the partition list. Please check if the following configs are present. I am attaching another boards defconfig for reference.

https://github.com/Samsung/TizenRT/blob/436a98058060741956e3b7bc347f96553549bcd3/build/configs/rtl8721csm/hello/defconfig#L266-L268